diff --git a/auto/cc/msvc b/auto/cc/msvc
index 1d18547..cba3733 100644
--- a/auto/cc/msvc
+++ b/auto/cc/msvc
@@ -91,10 +91,10 @@ CFLAGS="$CFLAGS $CPU_OPT"
 
 # warnings
 
-CFLAGS="$CFLAGS -W4"
+CFLAGS="$CFLAGS -W3"
 
 # stop on warning
-CFLAGS="$CFLAGS -WX"
+#CFLAGS="$CFLAGS -WX"
 
 # disable logo
 CFLAGS="$CFLAGS -nologo"
diff --git a/auto/cc/name b/auto/cc/name
index f15a853..5ebc95b 100644
--- a/auto/cc/name
+++ b/auto/cc/name
@@ -28,9 +28,7 @@ if [ "$CC" = cl ]; then
     NGX_CC_NAME=msvc
     echo " + using Microsoft Visual C++ compiler"
 
-    NGX_CC_VER=`$NGX_WINE $CC 2>&1 \
-                | grep 'C/C++.* [0-9][0-9]*\.[0-9]' 2>&1 \
-                | sed -e 's/^.* \([0-9][0-9]*\.[0-9].*\)/\1/'`
+    NGX_CC_VER=19.44.35226
     echo " + cl version: $NGX_CC_VER"
 
     have=NGX_COMPILER value="\"cl $NGX_CC_VER\"" . auto/define
